client/src/api/nftContract.ts

```diff
@@ -0,0 +1,70 @@
+import { TezosToolkit, MichelsonMap } from '@taquito/taquito';
+import { BigNumber } from 'bignumber.js';
+
+interface CreateTokenArgs {
+  ownerAddress: string;
+  symbol: string;
+  name: string;
+  description: string;
+  ipfsCid: string;
+}
+
+export interface NftContract {
+  createToken(args: CreateTokenArgs): Promise<number>
+}
+
+interface NftContractStorage {
+  next_token_id: string;
+}
+
+const mkNftContract =  async (
+  tzClient: TezosToolkit, 
+  address: string, 
+): Promise<NftContract> => {
+  const contract = await tzClient.contract.at(address);
+  
+  const constants = await tzClient.rpc.getConstants();
+  const pollingInterval: number = Number(constants.time_between_blocks[0]) / 5;
```

Comment:
We talked about setting polling interval when creating TezosToolkit, not for every operation

Reply:
We also talked that doin what is done here is not easy that way.